docs/pages/data_visualization_guidelines.js

```diff
@@ -0,0 +1,264 @@
+          <Flex direction="column" gap={8}>
+            <Flex direction="column" gap={1}>
+              <PaletteGenerator count={8} />
+            </Flex>
+            <Box marginBottom={10}>
+              <DataViz8Colors />
+            </Box>
+          </Flex>
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection
+        name="Pairings to avoid"
+        description="Although we recommend using colors following the order in the palette, we understand there might be edge cases when we need to pair colors in a different way. The color pairings below are hard to tell apart either in lines or small points under normal vision, or in large areas under red-green or yellow-blue color blindness. Always avoid using these pairings as neighboring colors."
+      >
+        <Flex gap={6} wrap>
+          <Flex direction="column" gap={6}>
+            <Flex direction="column" gap={1}>
+              <ColorTile
+                textColor="inverse"
+                description="Data Visualization 04"
+                fullTokenName="color-data-visualization-04"
+              />
+              <ColorTile
+                textColor="inverse"
+                description="Data Visualization 08"
+                fullTokenName="color-data-visualization-08"
+              />
+            </Flex>
+            <Flex direction="column" gap={1}>
+              <ColorTile
+                textColor="dark"
+                description="Data Visualization 01"
+                fullTokenName="color-data-visualization-01"
+              />
+              <ColorTile
+                textColor="inverse"
+                description="Data Visualization 11"
+                fullTokenName="color-data-visualization-11"
+              />
+            </Flex>
+
+            <Flex direction="column" gap={1}>
+              <ColorTile
+                textColor="dark"
+                description="Data Visualization 02"
+                fullTokenName="color-data-visualization-02"
+              />
+              <ColorTile
+                textColor="inverse"
+                description="Data Visualization 04"
+                fullTokenName="color-data-visualization-04"
+              />
+            </Flex>
+            <Flex direction="column" gap={1}>
+              <ColorTile
+                textColor="inverse"
+                description="Data Visualization 03"
+                fullTokenName="color-data-visualization-03"
+              />
+              <ColorTile
+                textColor="inverse"
+                description="Data Visualization 05"
+                fullTokenName="color-data-visualization-05"
+              />
+            </Flex>
+            <Flex direction="column" gap={1}>
+              <ColorTile
+                textColor="inverse"
+                description="Data Visualization 08"
+                fullTokenName="color-data-visualization-08"
+              />
+              <Box marginBottom={6}>
```

Comment:
is this marginBottom necessary?

Reply:
yeah it's for when it wraps into one column on mobile. Not the cleanest solution, but I'll add a comment

Reply:
👍 This is why I need to re-implement `gap` to include both column and row values 😬

Reply:
Yes please :P